examples/static-hbase/main.tf

```diff
@@ -106,8 +101,39 @@ module "emr-hbase" {
 
   # Spot Instance definition
   # on-demand r5.xlarge EC2 + EMR --> $0.252 + $0.063 = $0.315 per Hour
-  core_bid_price = ".100"
+  # core_bid_price = ".100"
 
   # Don't use spot instances for the master nodes
   master_bid_price = ""
```

Comment:
Recommendation is to not use spot for hbase, so we can remove these lines from the example.

Reply:
done